packages/server/graphql/handleGraphQLTrebuchetRequest.ts

```diff
@@ -51,16 +52,17 @@ const handleGraphQLTrebuchetRequest = async (
         safeError.stack = firstError.stack
         sendToSentry(safeError)
       }
-      if (result.data && IGNORE_MUTATIONS.includes(Object.keys(result.data)[0])) return
+      // no client sink = no opId = no response expected
+      if (!opId) return
```

Comment:
this is the crux of the file change. client now tells server what responses to ignore :tada: